app/utils/dns.py

```diff
@@ -1,14 +1,14 @@
 import dns.resolver
 import dns.name
 import dns.rdatatype
-import os
 import random
+import os
 
-from typing import Iterator, NamedTuple
+from typing import Iterator, NamedTuple, List
 
 
 resolver = dns.resolver.Resolver(configure=False)
-resolver.nameservers = [os.getenv("DNS_ADDR", "172.31.1.88")]
+resolver.nameservers = [os.environ.get("DNS_ADDR")]
```

Comment:
Why were all these `os.getenv` changed to `os.environ.get`?

Reply:
https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get

TL;DR - If this stuff is not set, raise an exception.